components/build-service/base/kustomization.yaml

```
@@ -0,0 +1,4 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - https://github.com/konflux-ci/konflux-ci/konflux-ci/build-service/core?ref=df1ab193b2b9f6224a4c09a5a2ee1bc0f3a4ae57
```

```suggestion
- https://github.com/konflux-ci/konflux-ci/konflux-ci/build-service?ref=76645e425f8b999e20f18e496c791d87521679ff```

This resource is referencing to `build-service/core`, but we need to refer to the root kustomization file which already contains the `core` resource